assets/feature-installer-utils.sh

```diff
@@ -1,4 +1,31 @@
-# Execute a command and if it fails, return its output and exit
+# Set some informative variables
+
+# The flavour we're running on
+FLAVOUR="$(cat /home/cloudcontrol/flavour)"
+export FLAVOUR
+
+# The path to install software binaries to
+BINPATH="/home/cloudcontrol/bin"
+export BINPATH
+
+TEMPDIR=""
+export TEMPDIR
+
+# Prepare feature installation. Will create a temporary directory and change to it
+function prepare {
+  TEMPDIR=$(mktemp -d)
+  cd "${TEMPDIR}" || exit
+}
+
+# Cleanup the previously generated temporary directory
+function cleanup {
```

Comment:
See prepare, `removeTempDir` or `cleanupTempDir` would fit and render comments superfluent

Reply:
See prepare